sdk/src/client/node_api/core/mod.rs

```diff
@@ -33,14 +33,12 @@ impl Client {
 
     /// Requests outputs by their output ID in parallel, ignoring failed requests.
     /// Useful to get data about spent outputs, that might not be pruned yet.
-    pub async fn get_outputs_ignore_errors(&self, output_ids: &[OutputId]) -> Result<Vec<Output>> {
-        Ok(
-            futures::future::join_all(output_ids.iter().map(|id| self.get_output(id)))
-                .await
-                .into_iter()
-                .filter_map(Result::ok)
```

Comment:
Shouldn't we instead filter only `crate::client::node_api::error::Error::NotFound(_)` and return other errors?

Reply:
Maybe, but they should be called _ignore_not_found then

Reply:
Yeah fine to have them renamed, but the idea was to just not fail if some outputs were already pruned. If it fails for another reason, we should still return an error I think

Reply:
Ok I see, that's not what they were currently doing, they were never failing atm.

Reply:
Probably also fine like this in 99,9% of the cases, but there is a small change that it could be a problem so we should change it

Reply:
Changed